public/app/features/provisioning/RepositoryStatusPage.tsx

```diff
@@ -1,14 +1,131 @@
+import { useMemo, useState } from 'react';
+import { useParams } from 'react-router-dom-v5-compat';
+
+import {
+  CellProps,
+  Column,
+  EmptyState,
+  FilterInput,
+  InteractiveTable,
+  LinkButton,
+  Spinner,
+  Stack,
+  Text,
+  TextLink,
+} from '@grafana/ui';
 import { Page } from 'app/core/components/Page/Page';
 
+import { useGetRepositoryStatusQuery, useListRepositoryFilesQuery } from './api';
+import { PROVISIONING_URL } from './constants';
+
 export default function RepositoryStatusPage() {
+  const { name = '' } = useParams();
```

Comment:
First time seeing this syntax: what's it do? Set the `name` to `''`, or just default it?

Reply:
It defaults the name to an empty string, but it's mostly to satisfy the types, since this page won't be rendered if the name is not there.